app/controllers/users_controller.rb

```diff
@@ -0,0 +1,21 @@
+class UsersController < ApplicationController
+    
+    def new
+      @user = User.new
+    end
+
+    def create
+      @user = User.new(user_params)
+      if @user.save
+          redirect_to "/"
```

Comment:
You might want to use the rails route helper. Then if the root path changes you dont need to update you controller.